fv/workload/workload.go

```diff
@@ -492,5 +500,5 @@ func (c *ConnectivityChecker) CheckConnectivityWithTimeout(timeout time.Duration
 		strings.Join(actualConn, "\n    "),
 		strings.Join(expConnectivity, "\n    "),
 	)
-	Fail(message, 1)
+	Fail(message, callerSkip)
```

Comment:
Note: I think default `callerSkip` here has changed from 1 to 2 - but I presume that that is what you want (for nicest debugging/reporting).

Reply:
Yes, exactly, I want it to report the failure on the test case line, not in `workload.go`.